server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesFetcher.java

```diff
@@ -234,6 +235,15 @@ private static boolean checkIncludeParents(String[] filters) {
         return true;
     }
 
+    private static boolean checkIncludeDimensions(String[] filters) {
+        for (String filter : filters) {
+            if ("+dimension".equals(filter)) {
```

Comment:
Nit: `+dimensions` reads better to me?  `+dimension` implies that there is only one.

Reply:
Although having said that, we already have `+parent` that can return multiple parents, so maybe it's fine.